### PR TITLE
Expansion of HIV diagnosis documentation

### PR DESCRIPTION
**Pull request by pineapple-cat**
Added module overview, data variables, and picked out a few questions to ask.

### CODE REVIEW

File: tutorials/hiv_diagnosis.md [hunk 1]
```diff
@@ -8,3 +8,29 @@ The HIV diagnosis module tracks the current HIV test type, test sensitivities, a
...
+
+Some people have a chance to disengage with medical care after diagnosis, which is represented by determining loss of care outcomes. Loss of care is affected by sex worker status in both primary infection and general populations, but it is also affected by the presence of short-term partners and HIV-adjacent diseases (ADC, TB, non-TB WHO3) in the general population only.
+
+### HIV Diagnosis Data Variables
+
+- *`hiv_test_type`* - The type of HIV test being administered; one of: Ab (default), NA, Ag/Ab.
+- *`init_prep_inj_na`* - A boolean randomly chosen by a coin flip at the start of the simulation to determine whether `prep_inj_na` has a chance to be True.
+- *`prep_inj_na`* - A boolean randomly chosen by a coin flip at the start of the simulation if `init_prep_inj_na` is True to determine whether NA tests should be used for those on injectable PrEP in the general population.
```
Comment (pineapple-cat) [inline tutorials/hiv_diagnosis.md:22]: Hi @UCL/hiv-modelling, should this `prep_inj_na` variable affect the value of `hiv_test_type`, the test type used for people in primary infection, or is the test type used for the general population completely separate?

Comment (andrew-phillips-1) [reply tutorials/hiv_diagnosis.md:22]: Hi @pineapple-cat No you are right.  We would only consider doing a nucleic acid (NA) based test in people starting or continuing inj (cab) prep.

Comment (andrew-phillips-1) [reply tutorials/hiv_diagnosis.md:22]: We discussed how hivtest_type is relevant also for people not in primary infection.  By default, a value of  hivtest_type is set at the top of the code for the whole model run (although the test type to be used can be changed for individuals lower in the code).  

This is used to determine the sensitivity in primary infection and chronic infection as below.

if hivtest_type=1 then do; sens_primary=0.86; sens_vct=0.98; spec_vct=1;     end; 
else if hivtest_type=3 then do; sens_primary=sens_primary_testtype3; sens_vct=0.98; spec_vct=0.992; end;
else if hivtest_type=4 then do; sens_primary=0.75; sens_vct=0.98; spec_vct=1; end;

Comment (pineapple-cat) [reply tutorials/hiv_diagnosis.md:22]: In the current iteration of the Python code `hiv_test_type` only ever affects test sensitivity for people in primary infection; it's worth revisiting the code at some point to find what I missed during the first pass of writing the diagnosis section.

File: src/hivpy/hiv_diagnosis.py [hunk 2]
```diff
@@ -34,7 +34,8 @@ def __init__(self, **kwargs):
         # FIXME: consider how to move more variables to data file
         self.hiv_test_type = HIVTestType.Ab
         self.init_prep_inj_na = rng.choice([True, False])
-        self.prep_inj_na = rng.choice([True, False]) if self.init_prep_inj_na else 0
+        # FIXME: should this variable affect the value of hiv_test_type or is it completely separate?
```
Comment (mmcleod89) [inline src/hivpy/hiv_diagnosis.py]:
```suggestion
```

**Review by mmcleod89 — APPROVED** [review]

**Pull request merged**